readthedocs/support/front.py

```diff
@@ -0,0 +1,31 @@
+"""Front's API client."""
+
+import requests
+
+
+class FrontClient:
```

Comment:
This seems like overkill for 3 API calls. Is there a reason that we're using a wrapper instead of just calling the API directly?

Reply:
we need to pass the token header on each request, felt like a lot of duplication, I'm fine moving this class to the views file. Can also merge this into the view class, but not a big fan of fat-views.